ui/src/app/cohort-search/list-search/list-search.component.tsx

```diff
@@ -16,15 +17,15 @@ const styles = reactStyles({
     position: 'absolute',
```

Comment:
Is all of this styling unique to this component? Or could some of it be inherited or pulled in from a common place?

Reply:
Yes, could use a refactor on several components to separate out common styles. Will try to do a follow-up pr with that.